business_requirement_from_support/models/project_issue.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# © 2017 Praxya (https://www.praxya.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models, _
+
+
+class ProjectIssue(models.Model):
+    _inherit = "project.issue"
+
+    business_requirement_id = fields.Many2one(
```

Comment:
Side note: what if the button is clicked 2 times? What would be the expected behavior?